source/slang/glsl.meta.slang

```diff
@@ -4985,9 +4985,9 @@ ${{{{
 }
 }}}}
 
-// all atomic_uint functions are HEAVILY mangled at compile time, 
-// all types are converted into 'inout uint' to the referenced 
-// layout(offset) of the related inputted atomic_uint
+// all atomic_uint functions are mangled at compile time, 
+// all types are converted into a field address of a 'uint'
+// relative to the layout(offset) of the atomic_uint 
```

Comment:
There is a trailing white character at the end of the line.
Please remove it.

Reply:
I don't think this was addressed(?)